docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md

```diff
@@ -81,7 +93,28 @@ caps.version = 67
 driver = Selenium::WebDriver.for :remote, :url => "http://www.example.com", :desired_capabilities => caps
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder, Capabilities } = require("selenium-webdriver");
+let chromeCapabilities = Capabilities.chrome();
+//Setting chrome options
+const capabilityName = 'goog:chromeOptions';
+var browserOptions = {
```

Comment:
Do we need these options for the example? It makes it more complex and harder to understand. The other examples do not show that.

Reply:
Hi @diemol 
Again I made change options calling. Hope now easy to understand.